mantidproject/mantid
# Indirect BayesQuasi - Documentation should now be able to be viewed on all OS

It should be possible to build the documentation for BayesQuasi on any OS which should mean that the documentation builds correctly for the web

**To test:**
- Ensure all tests pass for the algorithm (doc tests and unit tests)
- Build the documentation locally and check that it is correct (has all the algorithm information)
  - ideally this would be good to check on non-windows operating systems

Fixes #14646.

[Release Notes](https://github.com/mantidproject/mantid/blob/14646_BayesQuasi_documentation_no_showing/docs/source/release/v3.7.0/indirect_inelastic.rst#bugfixes)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
